contracts/state/PoseidonSMT.sol

```diff
@@ -130,6 +143,15 @@ contract PoseidonSMT is Initializable, UUPSUpgradeable {
         bytes32 root_ = _bytes32Tree.getRoot();
 
         IEvidenceRegistry(evidenceRegistry).addStatement(root_, bytes32(block.timestamp));
+        IMessageService(l2MessageService).sendMessage(
```

Comment:
Are we sending both certificate and registration roots to L1?

Reply:
No, added separate RegistrationSMT cotnract